scio-core/src/test/scala/com/spotify/scio/ArgsTest.scala

```diff
@@ -116,29 +116,29 @@ class ArgsTest extends AnyFlatSpec with Matchers {
 
   trait Options extends PipelineOptions {
     @Required
-    def getInput: String
-    def setInput(input: String): Unit
+    def getTestInput: String
+    def setTestInput(input: String): Unit
     @Required
-    def getOutput: String
-    def setOutput(output: String): Unit
```

Comment:
Since `PipelineOptions` are dynamically loaded, Some tests were failing because the `--input` and `--output` parameters were transformed into options instead of staying in the args. Renamed the values to avoid conflicts wit other tests